pulsar-flink/src/test/java/org/apache/flink/streaming/connectors/pulsar/PulsarSourceBuilderTest.java

```diff
@@ -112,6 +114,11 @@ public void testSubscriptionNameWithBlank() {
         pulsarSourceBuilder.subscriptionName(" ");
     }
 
+    @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
hmm, shouldn't this be NullPointerException?

Reply:
Yes, I wrote the wrong exception class